addons/website/views/snippets/s_features_wall.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_features_wall" name="Features Wall">
+    <section class="s_features_wall pt56 pb56" data-oe-shape-data="{'shape':'web_editor/Airy/04_001','colors':{'c1':'o-color-3'},'flip':'[x]','showOnMobile':false,'animated':'true'}">
+        <div class="o_we_shape o_web_editor_Airy_04_001 o_we_animated" style="background-image: url('/web_editor/shape/web_editor%2FAiry%2F04_001.svg?c1=%23E5EDF2&amp;flip=x');"/>
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="21">
+                <div class="o_grid_item g-height-21 g-col-lg-5 col-lg-5" style="z-index: 1; grid-area: 1 / 1 / 22 / 6; --grid-item-padding-y: 0px; --grid-item-padding-x: 16px;">
+                    <h2 class="display-1">Unveil Our Exclusive Collections</h2>
```

Comment:
Hey 👋 Thanks for the review!

You are right, the combination `h2` + `display-1` is not doable in the editor.

That said, the user can still implement the exact same result, visually speaking, by selecting the "Heading 1 - Display 1" text style. It's just that we can't use `h1` here (even if that would make it editor compliant AND would achieve the design we want), because it's a bad practice to have multiple `h1` and we already decided to use `h1` only in intro snippets. (Forcing the user to add an extra `h1` each time he wants to select a "display-x" font-style should be reconsidered too IMO)

Reply:
> Hey 👋 Thanks for the review!
> 
> You are right, the combination `h2` + `display-1` is not doable in the editor.
> 
> That said, the user can still implement the exact same result, visually speaking, by selecting the "Heading 1 - Display 1" text style. It's just that we can't use `h1` here (even if that would make it editor compliant AND would achieve the design we want), because it's a bad practice to have multiple `h1` and we already decided to use `h1` only in intro snippets. (Forcing the user to add an extra `h1` each time he selects a "display-x" font-style should be reconsidered too IMO)

I agree with you that it's too bad we cannot associate `h2` and `display-*`.
But it is what was decided, so we have to deal with it for now.
I asked QSM about your snippet, and he confirmed that we cannot introduce anything not doable with the editor.
So I'm sorry, but you might have to slightly change the design to achieve that.

Reply:
Okay, I guess we will have to deal with it now. I still hope we will see a font-weight controller in the web editor soon. 🤞